OnlineCinema/OnlineCinema/Modules/Home/HomeAssembler.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  HomeAssembler.swift
+//  OnlineCinema
+//
+//  Created by Лев Бондаренко on 11.03.2024.
+//
+
+import SwiftUI
+
+class HomeAssembler {
```

Comment:
What about using `enum` instead of `class` for assembler?
```suggestion
enum HomeAssembler {
```